### PR TITLE
xiaomi-mido: Add UCM configuration for xiaomi-mido

### PR DESCRIPTION
**Pull request by Kiciuk**
Speaker is connected to Diangu codec Lineout via an external amplifier.

### CODE REVIEW

**Comment by Minecrell** [comment]
Can you link to the (mainline) kernel source for these changes?

**Comment by Kiciuk** [comment]
https://github.com/msm8953-mainline/linux/commit/509ef3f26faa640cd4d8c637840892f0d5e4a1fb
Device tree for mido for now still contains msm8916 card but it will be changed once proper conf is available in package.

File: ucm2/xiaomi-mido/HiFi.conf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        Comment "Speaker playback"
+
+        ConflictingDevice [
+                "Headphones"
+                "Earpiece"
+        ]
+
+        EnableSequence [
+                cset "name='LINEOUT' 1"
+                cset "name='LINEOUT Ext' 1"
```
Comment (Minecrell) [inline ucm2/xiaomi-mido/HiFi.conf:23]: I think you should be able to remove this if you change in `audio-routing`:

```
"Speaker Amp IN", "LINEOUT_OUT",
```

then setting `LINEOUT` to `1` like you do one line above should be sufficient. I don't see any reason for adding the special Ext stuff here.

**Comment by Minecrell** [comment]
Pushed, thanks!

**Pull request closed without merging**